go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/buger/jsonparser v1.1.1
 	github.com/chzyer/readline v1.5.1
 	github.com/forPelevin/gomoji v1.1.6
+	github.com/gen2brain/beeep v0.0.0-20220909211152-5a9ec94374f6
```

Comment:
What is beeep? Please try to not add another dependency in this project

Reply:
beeep package is used for sending cross-platform desktop notifications

Reply:
Why do we need it? Can these commands be like the other CLI commands?

Reply:
Is there any existing functionality where JFrog CLI can send notifications to the desktop?

Reply:
@yahavi let me remove this for now.